package.json

```diff
@@ -6,6 +6,7 @@
   "browser": "browser.js",
   "typings": "index.d.ts",
   "scripts": {
+    "install": "node-gyp-build",
```

Comment:
What? Won't this always build?

Reply:
No because this module only builds if there isn't a pre-existing build or prebuild. Since we ship the prebuilt binaries, it will only build for unsupported platforms (i.e. ARM).

Reply:
It also supports the `--build-from-source` flag so we don't have to handle it as well.